new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -152,6 +153,30 @@ class LogFileManager {
         this.#decoder.setDecoderOptions(options);
     }
 
+    loadChunk (eventIdx: number): {
+        logs: string,
+    } {
+        const results = this.#decoder.decode(
+            eventIdx,
+            Math.min(eventIdx + EXPORT_LOGS_CHUNK_SIZE, this.#numEvents)
+        );
+
+        if (null === results) {
+            throw new Error("Error occurred during decoding chunk. " +
+                `eventIdx=${eventIdx});`);
+        }
+
+        const messages: string[] = [];
+        results.forEach((r) => {
+            const [msg] = r;
+            messages.push(msg);
+        });
```

Comment:
Can this be simplified as 
```suggestion
        const messages = results.map(([msg]) => msg);
```